client/src/components/content/RobotContainer/RobotContainer.jsx

```diff
@@ -15,16 +16,30 @@ const { Title } = Typography;
  * @category Client
  */
 const RobotContainer = (props) => {
-  const { robotId, robotName } = props;
+  const { robotId, robotName, userId } = props;
   const [name, setRobotName] = useState(robotName);
 
-  const startRobot = () =>
-    alert('Running the Robot is currently not supported!');
+  /**
+   * @description Sends a job to the server to execute a specfic robot for a specific user
+   */
+  const startRobot = () => {
+    // isBotExecutable funtion is to be implemented
```

Comment:
Just one question: at this point, Issue #161  must follow, right?

Reply:
Yes, exactly

Reply:
As far as my understanding goes, yes.